src/app/core/navbar/navbar.component.html

```diff
@@ -54,3 +59,37 @@
   </div>
 </nav>
 
+<!-- Changelog modal -->
+<div class="modal fade" id="changelog" tabindex="-1" role="dialog" aria-labelledby="changelogLabel" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered modal-lg">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="changelogLabel">Surprise! New Features!</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body" id="changelog-body">
+        <p>New features, fixes, and updates will be posted here whenever we update the website!</p>
+        <p><small>Note: Whenever we release an update to the 'signing' page, we may, at our discretion, choose to give
+          you a chance to send <em>another</em> message to each of your friends' virtrolios! This ensures that you will
+          be able to test out any new features that we develop. Your first message will
+          remain on your friend's virtrolio if you send a second one in this case, but if they delete it then you can no
+          longer send more than one message. Keep an eye on this changelog as we will mention here if we ever do give
+          you another message opportunity.</small></p>
```

Comment:
This is too tall. It takes up most of the screen in the modal on mobile - I would either put this elsewhere or make it shorter/smaller.